javaparser-core/src/main/java/com/github/javaparser/ast/Modifiers.java

```diff
@@ -0,0 +1,131 @@
+package com.github.javaparser.ast;
+
+import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.NodeWithModifiers;
+
+import javax.lang.model.element.Modifier;
```

Comment:
I disagree about that! I'm really not sure that we want to introduce a dependency on jx.l.e.
In fact, there is already a ref on j.l.reflect that is crazy and that we should remove.